types/abortable-iterator/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'abortable-iterator';
```

Comment:
Can you add a `ts-ignore` instead with a comment mentioning no types are exported? this will help us tracking better what modules that do not have types

Reply:
👍